src/cbmc/cbmc_parse_options.cpp

```diff
@@ -310,7 +322,8 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
 
   if(cmdline.isset("smt2"))
   {
-    options.set_option("smt1", false);// If both are given, smt2 takes precedence
+    // If both are given, smt2 takes precedence
+    options.set_option("smt1", false);
     options.set_option("smt2", true);
     version_set = true;
```

Comment:
I have moved comment to #343 https://github.com/diffblue/cbmc/pull/343/files#diff-41db988013f70caad45ce489749c7df6R327